source/screens/Home/Header/Header.tsx

```diff
@@ -5,15 +5,22 @@ import logo from '../../../assets/images/logo.png';
 import useWalletBalance from '../../../hooks/useWalletBalance';
 import useWalletSyncProgress from '../../../hooks/useWalletSyncProgress';
 import moneyFormatter from '../../../utils/moneyFormatter';
+import useFiatConversionRate from '../../../hooks/useFiatConversionRate';
 
 const Header = () => {
   const {balance} = useWalletBalance();
+  const {fiatRate} = useFiatConversionRate();
   const {initialized} = useWalletSyncProgress();
+  const {formatMoneyToString, formatDash} = moneyFormatter;
 
   const balanceFormatted = useMemo(() => {
-    return moneyFormatter.formatDuffs(balance);
+    return formatMoneyToString(formatDash(balance));
```

Comment:
I think it worth explicitly providing 8 as a second argument, otherwise you might end up with a longer string displayed